ksm-throttler/update_throttler.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+#
+# Automation script to create specs to build ksm-throttler.
+# Default: Build is the one specified in file configure.ac
+# located at the root of the repository.
+set -e
+
+source ../versions.txt
+source ../scripts/pkglib.sh
+
+SCRIPT_NAME=$0
+SCRIPT_DIR=$(dirname $0)
+PKG_NAME="ksm-throttler"
+VERSION=$ksm_throttler_version
+HASH=$ksm_throttler_hash
+RELEASE=$(cat release)
+
+BUILD_DISTROS=(Fedora_26 xUbuntu_16.04)
```

Comment:
Why only these two distros OOI? Also, I think we can potentially refactor those values into a config file or similar as they are used in quite a few scripts:

```
$ grep -iIrn BUILD_DISTROS|grep = 
kernel/update_kernel.sh:16:BUILD_DISTROS=(Fedora_26 xUbuntu_16.04)
shim/update_shim.sh:18:BUILD_DISTROS=(Fedora_26 xUbuntu_16.04)
proxy/update_proxy.sh:18:BUILD_DISTROS=(Fedora_26 xUbuntu_16.04)
qemu-lite/update_qemu.sh:18:BUILD_DISTROS=(Fedora_26 xUbuntu_16.04)
clear-containers-image/update_image.sh:19:BUILD_DISTROS=(Fedora_26 xUbuntu_16.04)
runtime/update_runtime.sh:20:BUILD_DISTROS=(Fedora_26 xUbuntu_16.04)

```

Reply:
Those were initially used for local build testing as the base for RPM and DEB packages, I can add the rest. And yeah, it will be better to have those in a file, thanks @jodh-intel!